sdk/include/opentelemetry/sdk/logs/processor.h

```diff
@@ -25,21 +25,44 @@ namespace sdk
 {
 namespace logs
 {
+enum class ShutdownResult
+{
+  kSuccess = 0,
```

Comment:
Make the make more specific like `kShutdownSuccess`? `enum class ShutdownResult` and `enum class ExportResult` live in the same namespace, and define the same enum field like `kSuccess` may cause confuse.

Reply:
Probably a boolean value would make it simpler?